qgispluginreleaser/entry_point.py

```diff
@@ -52,12 +64,13 @@ def fix_version(context):
     ``new_version``.
 
     """
-    if not prerequisites_ok():
+    metadata = metadata_file()
```

Comment:
Allebei is het eigenlijk metadata_file... Metadata, los, klinkt als ingelezen metadata.
De simpelste oplossing is de variabele `metadata_file` te noemen en de functie `find_metadata_file()`, denk ik?

Reply:
Ja, dat is wel mooier